src/Leaderboard.py

```diff
@@ -188,6 +188,11 @@ def showLeaderboard(player: str = None, limit: int = None) -> str or List[str]:
         return msgs if len(msgs) > 1 else msgs[0]
 
 
+def checkLeaderboard() -> bool:
+    if (len(leaderboard) < 6):
```

Comment:
I think it would be better to do `leaderboard.count()` here. There may be some optimizations that TinyDB does behind the scenes to make getting the count a bit faster. [https://tinydb.readthedocs.io/en/latest/api.html#tinydb.table.Table.count](https://tinydb.readthedocs.io/en/latest/api.html#tinydb.table.Table.count)

Reply:
Also, you should be able to just `return leaderboard.count() < 6`.

Reply:
Resolved.